src/items/generics.md

```diff
@@ -0,0 +1,107 @@
+# Type and Lifetime Parameters
+
+> **<sup>Syntax</sup>**  
+> _Generics_ :  
+> &nbsp;&nbsp; `<` _GenericParams_ <sup>?</sup> `>`  
+>  
+> _GenericParams_ :  
+> &nbsp;&nbsp; &nbsp;&nbsp; _LifetimeParams_  
+> &nbsp;&nbsp; | ( _LifetimeParams_ `,` ) <sup>?</sup>
+> _TypeParams_ `,` <sup>?</sup>  
+>  
+> _LifetimeParams_ :  
+> &nbsp;&nbsp; _LifetimeParam_ (`,` _LifetimeParam_)<sup>\*</sup>  
+>  
+> _LifetimeParam_ :  
+> &nbsp;&nbsp; [LIFETIME_OR_LABEL] [_LifetimeBounds_]<sup>?</sup>  
```

Comment:
You want a `&nbsp;` or "\ " (I think?) between the links or else markdown is going to think the first link is actually the text for the second link.